src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -2152,7 +2152,15 @@ let%test_module "test" =
       end
 
       module Sparse_ledger = struct
-        type t = int [@@deriving sexp, bin_io]
+        module Stable = struct
+          module V1 = struct
+            type t = int [@@deriving sexp, bin_io]
```

Comment:
can add `version` to the deriving list

Reply:
I removed the Stable modules for Sparse_ledger since it's not necessary anymore.`type t` is bin_io'd though.

Reply:
It's test code, so I'm not too worried about it, but really, the module level types should not have `bin_io`. Not needed to get the PR finished, though.